runtime/src/lib.rs

```diff
@@ -187,10 +188,20 @@ pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
 pub const HOURS: BlockNumber = MINUTES * 60;
 pub const DAYS: BlockNumber = HOURS * 24;
 
-// Unit = the base number of indivisible units for balances
-pub const UNIT: Balance = 10_000_000_000;
-pub const MILLIUNIT: Balance = UNIT / 1000;
-pub const MICROUNIT: Balance = MILLIUNIT / 1000;
+mod currency {
+	use super::Balance;
+
+	// Unit = the base number of indivisible units for balances
+	pub const UNIT: Balance = 10_000_000_000;
+	pub const MILLIUNIT: Balance = UNIT / 1000;
+	pub const MICROUNIT: Balance = MILLIUNIT / 1000;
+
+	pub const fn deposit(items: u32, bytes: u32) -> Balance {
+		items as Balance * 2 * UNIT + (bytes as Balance) * 100 * MILLIUNIT
```

Comment:
I propose you to leave it like this:

```suggestion
		items as Balance * 2 * UNIT + (bytes as Balance) * 300 * MICROUNIT
```